lighthouse-core/lib/i18n/en-US.json

```diff
@@ -0,0 +1,1858 @@
+{
+  "lighthouse-core/audits/accessibility/accesskeys.js | description": {
```

Comment:
this seems like a mistaken file to commit or bad merge or something @connorjclark ?

Reply:
> this seems like a mistaken file to commit or bad merge or something @connorjclark ?

fixed